powerfulseal/policy/policy_runner.py

```diff
@@ -34,19 +34,21 @@ class PolicyRunner():
         "scenarios": []
     }
 
-    def __init__(self, config_file, k8s_client, logger=None):
-        self.config_file = config_file
+    def __init__(self, policy_config, k8s_client, logger=None):
+        self.policy_config = policy_config
         self.k8s_client = k8s_client
         self.logger = logger or makeLogger(__name__)
 
     def read_policy(self):
         """
             Read configuration
         """
-        if self.config_file is None:
+        if isinstance(self.policy_config, str):
+            policy = PolicyRunner.load_file(self.policy_config)
+        elif isinstance(self.policy_config, dict):
```

Comment:
looks to me `policy_config` should be in `yml` format so maybe we do not need to support it as a `dict`?

Reply:
Another python process would send in a `dict` even if it were to parse a file as `yml` input. 

This is to support calling powerfulseal as a python library. In some cases we do not have control over the end host/container and don't want to inject the policy to disk.